src/setup-java.ts

```diff
@@ -1,8 +1,15 @@
 import * as core from '@actions/core';
 import * as installer from './installer';
 import * as auth from './auth';
+import * as gpg from './gpg';
 import * as path from 'path';
 
+const DEFAULT_ID = 'github';
+const DEFAULT_USERNAME = 'GITHUB_ACTOR';
```

Comment:
Some of these defaults can go straight into `action.yml`

Example: https://github.com/actions/setup-python/blob/654aa00a6ed0d2f7cf168f68bf3cca5daf06cded/action.yml#L13